test/network/BancorNetwork.ts

```diff
@@ -1383,6 +1477,232 @@ describe('BancorNetwork', () => {
             await expect(network.withdraw(12_345)).to.be.revertedWithError('AccessDenied');
         });
 
+        it('should revert when attempting to withdraw surplus tokens for invalid pools', async () => {
+            await expect(network.withdrawSurplusTokens(ZERO_ADDRESS)).to.be.revertedWithError('InvalidToken');
+        });
+
+        it('should revert when attempting to withdraw surplus tokens for pools with enabled trading', async () => {
```

Comment:
should also test for a non whitelisted pool

Reply:
added non whitelisted pool test